crest/Assets/Crest/Crest/Scripts/LodData/LodDataMgrDynWaves.cs

```diff
@@ -38,7 +38,7 @@ public override SimSettingsBase CreateDefaultSettings()
         static int sp_Damping = Shader.PropertyToID("_Damping");
         static int sp_Gravity = Shader.PropertyToID("_Gravity");
         static int sp_LaplacianAxisX = Shader.PropertyToID("_LaplacianAxisX");
-
+        
```

Comment:
Lingering whitespace, might be worth doing a format on all of the changed files (I'm happy to do so if it saves time).

Reply:
I've done it